src/Agent.Worker/TestResults/NunitResultReader.cs

```diff
@@ -320,6 +327,14 @@ private TestCaseResultData getTestCaseResultData(XmlNode testCaseResultNode, str
                     var failureStackTraceNode = failureNode.SelectSingleNode("stack-trace");
                     testCaseResultData.ErrorMessage = failureMessageNode?.InnerText;
                     testCaseResultData.StackTrace = failureStackTraceNode?.InnerText;
+
+                    // console log
+                    XmlNode consoleLog = testCaseResultNode.SelectSingleNode("output");
+                    if (consoleLog != null && !string.IsNullOrWhiteSpace(consoleLog.InnerText))
+                    {
+                        testCaseResultData.ConsoleLog = consoleLog.InnerText;
+                    }
```

Comment:
I don't think we need this as of now. This story is to capture stdout/stderr logs only for failed tests. This is applicable for other scenarios as well.

Reply:
Synced offline this is for handling other failure call flow in NUnit.